requirements-dev.txt

```diff
@@ -9,3 +9,4 @@ ruamel.yaml
 beautifulsoup4
 black
 jsonschema
+urllib3<2
```

Comment:
What problem are we solving with this?

Reply:
I suggest a comment, ideally with a poo# reference, explaining why to avoid somebody being smart and undoing whatever you're fixing since we're already getting the dep elsewhere.

Reply:
Is there an issue for the responses module maybe that we could link to?

Reply:
So the problem has already been resolved by upstream but it hasn't been released yet.

Reply:
I would still mention that here.